lib/jsdom/living/file-reader.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+const DOMException = require("../web-idl/DOMException");
+const EventTarget = require("./generated/events/EventTarget");
+
+const blobSymbols = require("./blob-symbols");
+
+function FileReaderEventTarget() {
+  if (!(this instanceof FileReaderEventTarget)) {
+    throw new TypeError("DOM object constructor cannot be called as a function.");
+  }
+  EventTarget.setup(this);
+}
+
+FileReaderEventTarget.prototype = Object.create(EventTarget.interface.prototype);
+
+module.exports = function (core) {
```

Comment:
What do you think of a structure where instead of adding `createFileReader` to core and then deleting it from window, you return it? So the usage code would be

```js
const createFileReader = require("../living/file-reader");

...

this.FileReader = createFileReader(this);
```

(and living/index.js would not use file-reader.js)

What you have now is OK too thought if you think that's better.